src/events/Windows.cpp

```diff
@@ -373,6 +376,24 @@ void Events::listener_mapWindow(void* owner, void* data) {
                         }
                     }
 
+                    if (ONSCREEN) {
+                        int borderSize = PWINDOW->getRealBorderSize();
+
+                        // left
+                        if (posX < PMONITOR->vecReservedTopLeft.x + borderSize)
```

Comment:
wait, why not use `std::clamp` ?

Reply:
I don't expect you to remember, but I did mention I'm new to C++... Guess I'll start looking in the stdlib for everything and the kitchen sink.   Is this appropriate?

```
posX = std::clamp(posX, int(PMONITOR->vecReservedTopLeft.x + borderSize),
    int(PMONITOR->vecSize.x - PMONITOR->vecReservedBottomRight.x - PWINDOW->m_vRealSize.goalv().x - borderSize));

posY = std::clamp(posY, int(PMONITOR->vecReservedTopLeft.y + borderSize),
    int(PMONITOR->vecSize.y - PMONITOR->vecReservedBottomRight.y - PWINDOW->m_vRealSize.goalv().y - borderSize));

```

Reply:
yep, looks about right.

just `(int)(something)`, because `int(something)` hurts in the eyes.

> I don't expect you to remember, but I did mention I'm new to C++...

No shame in that, I am just giving tips. In the case of you not using `std::clamp` because of something, I asked the question the way I did.